packages/application-components/src/components/internals/for-docs/example-wrapper/example-wrapper.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ThemeProvider } from 'emotion-theming';
+import { IntlProvider } from 'react-intl';
+import { customProperties } from '@commercetools-frontend/ui-kit';
+import ModalController from '../modal-controller';
+
+const ExampleWrapper = props => (
+  <ThemeProvider
+    theme={{
+      // Reset theme to default styles, so that the example uses the default uikit tokens
+      fontFamilyDefault: customProperties.fontFamilyDefault,
+      colorSolid: customProperties.colorsSolid,
```

Comment:
By resetting the custom style themes of the website, we ensure that the example components use the default uikit/mc styles.

Reply:
yeah this makes sense 👍